src/BenchmarksApps/HttpClientBenchmarks/Servers/Kestrel/Program.cs

```diff
@@ -80,17 +94,29 @@ public static void RunKestrel()
 
         app.MapGet("/", () => Results.Ok());
 
-        app.MapGet("/get", () => "Hello World!");
+        app.MapGet("/get/{responseSize}", GetResponse);
 
-        app.MapPost("/post", async (HttpRequest request) =>
+        app.MapPost("/post/{responseSize}", async (HttpRequest request, string responseSize) =>
         {
             await request.Body.CopyToAsync(Stream.Null);
-            return Results.Text("Server received the content");
+            return GetResponse(responseSize);
         });
 
         app.Run();
     }
 
+    private static IResult GetResponse(string responseSize)
```

Comment:
Instead of hardcoding 4 different options, can we match what the default downstream program does to enable arbitrary values?
https://github.com/aspnet/Benchmarks/blob/d5f47088cd6582ad35b44ffb5ef93c7c049bf822/src/Downstream/Program.cs#L86-L90

Reply:
Right, a single buffer, initialized from the argument. Also does the content in the buffer matter? (randome seed).

Reply:
Yep, that was the other option in my mind. I don't feel strongly on one vs another. The content doesn't matter. I'll make the change